todo_app/app.py

```diff
@@ -114,7 +125,6 @@ def add_Task():
     def get_Task():
 
         taskId = request.args.get("taskId")
-        # task = getItem(collection=todo.todo, taskId=taskId)
         task = app_db.get_item(taskId)
 
         return render_template("task.html", task=task, taskId=task.id, user=app_user)
```

Comment:
Minor, but there's one remaining reference to `app_user` here that we want to swap out:
```suggestion
        return render_template("task.html", task=task, taskId=task.id, user=current_user)
```